examples/seismic/model.py

```diff
@@ -490,22 +502,36 @@ def domain_size(self):
 class Model(GenericModel):
     """The physical model used in seismic inversion processes.
 
-    :param origin: Origin of the model in m as a tuple in (x,y,z) order
-    :param spacing: Grid size in m as a Tuple in (x,y,z) order
-    :param shape: Number of grid points size in (x,y,z) order
-    :param space_order: Order of the spatial stencil discretisation
-    :param vp: Velocity in km/s
-    :param nbpml: The number of PML layers for boundary damping
-    :param epsilon: Thomsen epsilon parameter (0<epsilon<1)
-    :param delta: Thomsen delta parameter (0<delta<1), delta<epsilon
-    :param theta: Tilt angle in radian
-    :param phi: Asymuth angle in radian
+    Parameters
```

Comment:
@mloubout

Reply:
yes?

Reply:
can you check the data types below in the docstring? as well as the description

Reply:
and can you provide the missing data types (eg for ``origin``)

Reply:
Provided, resolved